galera/Project1/Galera.cpp

```diff
@@ -32,3 +32,65 @@ int Galera::getTotalSallary() const
 {
     return _totalSallary;
 }
+
+bool Galera::hireEmployee(Employee first)
+{
+    bool k = true;
+    if (_employeeList.empty())
+    {
+        _employeeList.push_back(first);
+        return true;
+    }
+    auto x = _employeeList.begin();
+    for (int i = 0; i < _employeeList.size(); i++)
+    {
+        
+        if (first == *x)
+        {
+            return false; 
+        }
+        ++x;
+    }
+    if (k == true)
+    {
+        _employeeList.push_back(first);
+        return true;
+    }
+}
+
+ ostream& operator<<(ostream & output,const Galera& object)
+{
+     output << "galera name is -" <<object._name <<" ";
+     output << "galera employee count  is -" << object._employeeCount << " ";
+     output << "galera total sallary is -" << object._totalSallary << endl;
+     output << "employee:" << endl;
+
+     for (auto employee :object._employeeList)
+     {
+         output << employee;
+     }
+
+     return output;
+}
+ 
+
+bool Galera::fireEmployee(Employee object)
+{
+    if (_employeeList.empty())
+    {
+        return false;
+    }
+   
+    for (int i = 0; i < _employeeList.size(); i++)
+    {
+        
+        if (object == _employeeList[i])
+        {
+            _employeeList.erase(_employeeList.begin() + (i));
+            return true;
+        }
+    }
+    
```

Comment:
In case when `_employeeList.empty() == false` but that employee isn't working inside this `Galera` then you function should return `false`

Reply:
@pavlo-onyshchak  check comment above please